### PR TITLE
docs: update README to reflect the current state of the project

### LINKED ISSUE #634: Update README
The README is wildly out of date 😸 Let’s fix that.

### PR DESCRIPTION
**Pull request by schalkneethling**
Update and improve the README for contributors

Fix #634

### CODE REVIEW

**Comment by schalkneethling** [comment]
Ugh! GPG is driving me bonkers 😄

File: README.md [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
-[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+Builder of Bits aka The [MDN Web Docs interactive examples](https://github.com/mdn/interactive-examples) example builder.
 
-Builder of Bits aka The [MDN Web Docs interactive examples](https://github.com/mdn/interactive-examples), example builder
+## Contributing
+
+If you want to contribute to BoB, please follow these steps:
+
+- [Create a fork](https://docs.github.com/en/get-started/quickstart/contributing-to-projects#forking-a-repository)
+- [Open and issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) if one does not [already exist](https://github.com/mdn/bob/issues)
```
Comment (caugner) [inline README.md]:
```suggestion
- [Open an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) if one does not [already exist](https://github.com/mdn/bob/issues)
```

**Review by caugner — APPROVED** [review]
LGTM, just one typo

**Comment by schalkneethling** [comment]
Thank you, @caugner 🎉

**Pull request merged**